torchci/clickhouse_queries/experiment_rollover_percentage/query.sql

```diff
@@ -72,15 +70,14 @@ WITH
         FROM
             success_stats AS experiment
         INNER JOIN
-            success_stats AS m ON experiment.bucket = m.bucket
+            success_stats AS m
+        ON
+            experiment.label_ref = m.label_ref
+            AND experiment.bucket = m.bucket
         WHERE
-            experiment.job_name = m.job_name
-            AND experiment.workflow_name = m.workflow_name
-            AND experiment.is_ephemeral_exp = 1 AND m.is_ephemeral_exp = 0
-            AND experiment.group_size > 3
```

Comment:
Confirming that this is fine to remove since we no longer use this query to compare the health of different fleets. 

The reason for this group_size > 3 check is was originally because when we were bringing the LF fleet on line we wanted to see if the LF fleet was more likely to fail jobs, hinting at some underlying problem, so we would compare the success rates of the same job on each fleet and expect the ratio to be ~1.  

However that when the % of jobs enabled or disabled was really high/low then a single failure would cause the job's statistics to diverge significantly, hence the filter.